<launch>
		<!-- Drone's physical characteristics /-->
		<arg name="a_max" default="5"/> 
		<arg name="v_max" default="2.5"/> 
		<arg name="planner_drone_radius" default="2"/>
		<arg name="planner_drone_height" default="0.6"/>

		<arg name="planning_budget" default="5"/>
		<param name="planning_budget" value="$(arg planning_budget)"/>

	<include file="$(find airsim_tutorial_pkgs)/launch/front_stereo_and_center_mono/front_stereo_and_center_mono.launch"/>
	
	<node pkg="airsim_ros_pkgs" name="package_delivery_static" type="package_delivery_static" output="screen" />

	<node pkg="airsim_ros_pkgs" name="motion_planner" type="motion_planner">
		<param name="motion_planning_core" value="OMPL_RRTConnect"/>
		<param name="sampling_interval" value="0.5"/>
        <param name="planner_drone_radius" value="$(arg planner_drone_radius)"/>
        <param name="planner_drone_height" value="$(arg planner_drone_height)"/>
        <param name="v_max" value="$(arg v_max)"/>
		<param name="a_max" value="$(arg a_max)"/>
	</node>

	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server" args="$(find airsim_ros_pkgs)/include/airsim_map.bt">
		<param name="resolution" value="0.6" />
		
		<param name="frame_id" type="string" value="world_enu" />
	</node>

</launch>